packages/data-point-cache/lib/cache.js

```diff
@@ -8,13 +8,26 @@ const DefaultSettings = {
   localTTL: ms('2s')
 }
 
+/**
+ * @param {String|Number} value
+ * @returns {Number}
+ */
+function normalizeMilliseconds (value) {
+  return typeof value === 'string' ? ms(value) : value
+}
+
 function set (cache, key, value, ttl = '20m') {
-  const ttlms = typeof ttl === 'string' ? ms(ttl) : ttl
+  const ttlms = normalizeMilliseconds(ttl)
   return cache.redis
     .set(key, value, ttlms)
     .then(res => cache.local.set(key, value, cache.settings.localTTL))
 }
 
+/**
+ * @param {Object} cache cache store
+ * @param {String} key cache key
+ * @returns {Promise<*|undefined>} returns value whenr esolved or undefined
+ */
 function getFromStore (cache, key) {
   return Promise.resolve(cache.local.get(key)).then(entry => {
     // if we still have it sotred locally return the value, skip everything else
```

Comment:
Small typo here: "sotred" instead of "sorted"

Reply:
Another small typo: "whenr esolved" instead of "when resolved"

Reply:
thanks! will fix